docs/development/extensions-core/kinesis-ingestion.md

```diff
@@ -471,3 +470,13 @@ with an assignment of closed shards that have been fully read and to ensure a ba
 This window with early task shutdowns and possible task failures will conclude when:
 - All closed shards have been fully read and the Kinesis ingestion tasks have published the data from those shards, committing the "closed" state to metadata storage
 - Any remaining tasks that had inactive shards in the assignment have been shutdown (these tasks would have been created before the closed shards were completely drained)
+
+## Kinesis known issues
+
+Before you deploy the Kinesis extension to production, consider the following known issues:
```

Comment:
Another known issue - 
Kinesis supervisor can sometimes check if the checkpointed offset has fallen behind the retention window of the stream. These checks fetch the earliest sequence number of kinesis which can result in `IteratorAgeMilliseconds` becoming very high in AWS cloudwatch.